sherpa-onnx/csrc/online-model-config.h

```diff
@@ -45,6 +45,10 @@ struct OnlineModelConfig {
   std::string modeling_unit = "cjkchar";
   std::string bpe_vocab;
 
+  /// if tokens_buf_str is non-empty, 
+  /// the tokens will be loaded from the buffered string in prior to the ${tokens} file
+  std::string tokens_buf_str;
```

Comment:
```suggestion
  std::string tokens_buf;
```

`tokens_buf` is shorter.